src/mcts/search.cc

```diff
@@ -956,7 +956,14 @@ SearchWorker::NodeToProcess SearchWorker::PickNodeToExtend(
         ++possible_moves;
       }
       const float Q = child.GetQ(fpu);
-      const float score = child.GetU(puct_mult) + Q;
+      float score = child.GetU(puct_mult);
+      if (Q == 0) {
+        continue;
```

Comment:
This doesn't look right.

Reply:
For `Q=0`, `Q+U = U`. Am I missing something?

Reply:
Yes, this should just be an empty bracket.

Reply:
Or just change the last `else` into `else if (Q != 0)` and remove the first block.